src/components/calculator.js

```diff
@@ -41,6 +42,7 @@ const Calculator = () => {
           <Button label="=" onClick={() => handleButtonClick('=')} />
         </div>
       </div>
+      <Quotes />
```

Comment:
- [ ] Great job for creating the component for displaying quotes :+1: In the project's requirements, you were asked to import the new component into the main component(App component). The component should be on the same page as the Calculator component. I think they are asking us to import this component into the App component as the Calculator component. Please let's kindly adhere to the requirements.
![Screenshot from 2023-06-03 22-25-17](https://github.com/Unleashedicon/Math-magicians-project/assets/63087491/a6c6eeb1-e1a6-4f8d-9da5-3a2554528eea)